### PR TITLE
Domain verification UX fixes

### PR DESCRIPTION
**Pull request by mattyza**
This pull request addresses a few UX items when using the new domain verification feature.

- When the secret key or test secret key aren't present, the "you've verified your domain" and "re-verify domain" buttons are displayed. This shouldn't display, if either of the secret keys are missing.
- T&C text and links should be in the "description" field.
- When I click "re-verify domain", I see no "verifying..." action, so I'm not sure anything is happening.

One additional question I have is, how do I force a verification error, to test the verification flow when it fails?

@roykho

### CODE REVIEW

**Comment by roykho** [comment]
Currently if you leave the secret key blank and you click on verify, it will let you know the key is missing. Doing it this way requires less code. Otherwise you will need logic in place using JS to constantly check the secret key field. Also I don't see this logic in your PR.

To cause a fail on verification, simply type in some random secret key.

File: assets/js/stripe-admin.js [hunk 1]
```diff
@@ -91,10 +91,10 @@ jQuery( function( $ ) {
 				e.preventDefault();
 
 				// Remove any previous messages.
-				$( '.wc-stripe-apple-pay-domain-message' ).remove();
+				$( '.wc-stripe-apple-pay-domain-message' ).text( '' );
```
Comment (roykho) [inline assets/js/stripe-admin.js]: It is not good practice to blank out a DOM element leaving it sit there hidden with no purpose. It is better to remove and add it when need it.

File: assets/js/stripe-admin.js [hunk 2]
```diff
@@ -91,10 +91,10 @@ jQuery( function( $ ) {
...
 
 				if ( ! wc_stripe_admin.getSecretKey() ) {
-					$( '#wc-gateway-stripe-apple-pay-domain' ).after( '<p class="wc-stripe-apple-pay-domain-message" style="color:red;">' + wc_stripe_admin_params.localized_messages.missing_secret_key + '</p>' );
+					$( '.wc-stripe-apple-pay-domain-message' ).attr( 'style', 'color: red;' ).text( wc_stripe_admin_params.localized_messages.missing_secret_key );
```
Comment (roykho) [inline assets/js/stripe-admin.js]: If you want to go this route with the styles, you might as well use `css()` method.

File: assets/js/stripe-admin.js [hunk 3]
```diff
@@ -114,14 +114,15 @@ jQuery( function( $ ) {
 					url:     wc_stripe_admin_params.ajaxurl,
 					success: function( response ) {
 						if ( true === response.success ) {
-							$( '#wc-gateway-stripe-apple-pay-domain' ).html( wc_stripe_admin_params.localized_messages.re_verify_button_text ).after( '<p class="wc-stripe-apple-pay-domain-message" style="color:green;">' + response.message + '</p>' );
+							$( '#wc-gateway-stripe-apple-pay-domain' ).html( wc_stripe_admin_params.localized_messages.re_verify_button_text );
+							$( '.wc-stripe-apple-pay-domain-message' ).attr( 'style', 'color: green;' ).text( response.message );
```
Comment (roykho) [inline assets/js/stripe-admin.js]: Use `css()` method.

File: includes/class-wc-gateway-stripe.php [hunk 4]
```diff
@@ -203,6 +203,7 @@ public function generate_apple_pay_domain_html() {
 			<td class="forminp">
 				<?php if ( ! empty( $this->secret_key ) && ! $this->apple_pay_domain_set ) { ?>
 					<a href="#" id="wc-gateway-stripe-apple-pay-domain" class="button button-secondary"><?php esc_html_e( 'Verify Domain', 'woocommerce-gateway-stripe' ); ?></a>
+					<p class="wc-stripe-apple-pay-domain-message" style="color:green;"></p>
```
Comment (roykho) [inline includes/class-wc-gateway-stripe.php]: Empty P is just not good.

**Comment by mattyza** [comment]
After a chat with Apple Pay yesterday, it looks like we'll be removing this UI, so no need for this PR anymore.

**Pull request closed without merging**